content/partner/aws.md

```diff
@@ -1,19 +1,109 @@
+        - header: Trust (and prove) your secrets are secure
+          body: Adopt dynamic, short-lived secrets on demand as a best practice. Lock down every environment with [role-based access controls](/docs/pulumi-cloud/admin/organizations/#organization-roles), [versioning](/docs/esc/environments/versioning/), and a full audit log of all changes.
+        - header: Ditch .env files
+          body: No more copying-and-pasting secrets or storing them in plaintext on dev computers. Easily access secrets via CLI, API, Kubernetes operator, the Pulumi Cloud UI, and SDKs.
+    - title: Manage
+      icon: testing
+      color: blue
+      subtitle: Gain full auditability and visibility into all your environments to tame complexity and put security first.
+      description: Manage cloud infrastructure with dashboards that provide visibility into your infrastructure and any changes, [role-based access controls](/docs/pulumi-cloud/admin/organizations/#organization-roles), and [Policy as Code](/docs/iac/packages-and-automation/crossguard/) enforcement across your organization.
+      image: /images/product/insights-resource-search.png
+      points:
+        - header: Converse about your infrastructure
+          body: |
+            [Pulumi Copilot](/product/copilot/) makes discovering cost savings, running compliance checks, and debugging deployments across your Kubernetes resources as easy as typing a question.
+        - header: Enforce compliance
+          body: Enable [Policy as Code](/docs/iac/packages-and-automation/crossguard/) within your organization so that you can define guardrails for your infrastructure, ensuring engineers are following best practices and putting security first. This helps you prevent mistakes before they occur and respond rapidly to any incidents.
+        - header: Clear visibility across AWS resources
+          body: Ask any questions about your AWS infrastructure. Pulumi Insights helps you find that needle in the haystack – locating a single resource across regions and accounts. See every resource running in each stack with deep links to the AWS Console, actions performed by team members, Git-like diffs for infrastructure changes, and much more.
+
+customers:
+    - name: Snowflake
+      link: /case-studies/snowflake
+      logo: /logos/customers/snowflake-logo.svg
+    - name: Mercedes-Benz Research and Development
+      link: /case-studies/mercedes-benz/
+      logo: /logos/customers/mercedes-benz-RDNA_logo.png
+    - name: MindBody
+      logo: /logos/customers/mindbody_logo.svg
+    - name: National Institutes of Health
+      logo: /logos/customers/nih.png
+    - name: BMW
+      link: /case-studies/bmw
+      logo: /logos/customers/bmw.svg
+    - name: Unity
+      link: /case-studies/unity
+      logo: /logos/customers/unity.png
+    - name: Starburst
+      link: /case-studies/starburst
```

Comment:
I think starburst shouldn't be the only one on the 2nd row. We should move like BMW and Unity down as well